app/views/panoptic/jobs/failed/show.html.erb

```diff
@@ -5,6 +5,8 @@
 
 <h1><%= @job.class_name %></h1>
 
+<%= button_to "Retry Job", retry_job_path(@job), method: :patch, class: "btn btn-primary mb-3" %>
```

Comment:
How do you feel about adding the action to the index page as well? On the job row?